docs/ibm-cloud-rules.md

```diff
@@ -795,7 +795,7 @@ To define a new rule as part of your custom ruleset, please read the [Spectral C
 Rather than turning off a Spectral rule entirely, Spectral overrides allow you to customize ruleset usage for different
 files and projects without having to duplicate any rules.
 For details on how to add overrides to your custom ruleset, please read the
-[Spectral overrides](https://meta.stoplight.io/docs/spectral/ZG9jOjI1MTg5-custom-rulesets#overrides) documentation.
+[Spectral overrides](https://meta.stoplight.io/docs/spectral/293426e270fac-overrides) documentation.
```

Comment:
It would be really great if the spectral folks could somehow maintain a static/constant link to their docs, but IIRC this link changes from time to time.